src/Core/Content/ImportExport/DataAbstractionLayer/Serializer/Entity/OrderSerializer.php

```diff
@@ -41,7 +41,7 @@ public function serialize(Config $config, EntityDefinition $definition, $entity)
             $entity['lineItems'] = implode('|', $modifiedLineItems);
         }
 
-        if (isset($entity['deliveries'])) {
+        if (isset($entity['deliveries']) && count($entity['deliveries']) > 0) {
```

Comment:
That looks like a broken order when it doesn't have any deliveries

Reply:
This addresses orders with just one line item which is a "credit". If these orders are supposed to have deliveries, this change can be rejected/closed.